shared/css/sidebar.css

```diff
@@ -0,0 +1,98 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
```

Comment:
This is the wrong license, we should be using one of the MPL headers here: https://www.mozilla.org/en-US/MPL/headers/